addons/website_theme_install/models/ir_module_module.py

```diff
@@ -236,7 +236,10 @@ def _theme_cleanup(self, model_name, website):
         if model_name in ('website.page', 'website.menu'):
             return model
 
-        orphans = model.with_context(active_test=False).search([
+        # use active_test to also unlink archived models, and use _force_unlink
+        # to also unlink inherited models
+        orphans = model.with_context(active_test=False, _force_unlink=True).search([
```

Comment:
What happened to MODULE_UNINSTALL_FLAG?
Could not use it outside the `base ir_model` file?

Reply:
I was hesitant because it wasn't used in other places but I think it's a good idea. So I'll say I was just waiting for you to ask before adding it.